.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: java
+
+jdk:
+  - oraclejdk8
+
+sudo: required
+dist: trusty
+
+services:
+  - docker
+
+before_install:
+  # This section can be removed once travis get equipped with docker 1.10 and docker-compose 1.6
```

Comment:
There are older versions installed in travis. I would prefer to remove this code and use whatever travis has, hence I stick with oldest possible versions of docker and docker-compose. I hope that waiting period for that version in travis should shorter with that approach.